modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx

```diff
@@ -108,11 +108,11 @@ export default function LeftSidebar({
 			};
 
 			try {
-				const newObjectDefinition = (await API.save(
-					`/o/object-admin/v1.0/object-definitions/${objectDefinition?.id}`,
-					movedObjectDefinition,
-					'PATCH'
-				)) as ObjectDefinition;
+				const newObjectDefinition = (await API.save({
```

Comment:
@igor-franca In this case, shouldn't we be passing the property `returnValue: true`?